### DISCUSSION TITLE
having problems with SONOFF Zigbee 3.0 USB Dongle Plus Gateway

### DISCUSSION THREAD

**Discussion by lumpynose** [Koenkk/zigbee2mqtt#11604] [Devices]
I have it plugged into a Raspberry Pi 4b.  Also plugged into the Pi is my boot drive, a USB Sandisk SSD.

Whenever I unplug the Zigbee dongle the disk becomes unavailable, or something.  All commands typed at the prompt fail; halt, reboot, etc.  The only option is to pull the power plug on the Pi and restart.  (I haven't tried unplugging and plugging in the SSD; seems safer to have the system restart and hopefully fix any filesystem errors.)

Is anyone else having this problem?

Is there a better Zigbee dongle I should be looking at?  I'm going to order the Conbee II since it has a lot of stars on amazon and hopefully its higher price means better reliability and quality.

**Comment by eglische** [comment]
Hello
Just a small Input, if you run the ssd and the dongle over the internal USB connection could case power issues.
check for under voltage, as i understand, it's best to run ssd and other stuff over powered external USB hub.

Maybe it helps
Cheers
Roger

**Reply by lumpynose** [reply]
Thank you.  That's a very good point.  I forgot about the power issues with an external drive.  Although I am using a 3 amp USB power adapter but even so it might be chancy with both of these things plugged into the Pi.  I have a Pi Zero 2 that's not doing anything; I should move the zigbee dongle over to it.
